TradingOnTalentURAP/matching.py

```diff
@@ -0,0 +1,567 @@
+"Duke":["Duke University"],
+"ECU":["East Carolina University"],
+"EIU":["Eastern Illinois University"],
+"EMU":["Eastern Mennonite University"],
+"EMU":["Eastern Michigan University"],
+"ENC":["Eastern Nazarene College"],
+"ETSU":["East Tennessee State University"],
+"EWU":["Eastern Washington University"],
+"EKU":["Eastern Kentucky University"],
+"F&M":["Franklin & Marshall College"],
+"FAMU":["Florida A&M University"],
+"FAU":["Florida Atlantic University"],
+"FC":["Founders College"],
+"FDU":["Fairleigh Dickinson University"],
+"FGCU":["Florida Gulf Coast University"],
+"FIU":["Florida International University"],
+"Foothill":["Foothill College"],
+"FPU":["Fresno Pacific University"],
+"Fresno State":["California State University, Fresno"],
+"FSU":["Fayetteville State University", "Ferris State University", "Fitchburg State University", "Florida State University", "Framingham State University", "California State University, Fresno", "Frostburg State University"],
+"FU":["Fordham University", "Furman University"],
+"GB":["University of Wisconsin–Green Bay"],
+"GC&SU":["Georgia College & State University"],
+"GCSU":["Georgia College & State University"],
+"GCU":["Grand Canyon University"],
+"Georgia College":["Georgia College & State University"],
+"George Mason":["George Mason University"],
+"GMU":["George Mason University"],
+"Georgia Tech":["Georgia Institute of Technology"],
+"George Washington":["George Washington University"],
+"GW":["George Washington University"],
+"GWU":["George Washington University"],
+"GGC":["Georgia Gwinnett College"],
+"GONZ":["Gonzaga University"],
+"GSU":["Georgia Southern University", "Georgia State University", "Grambling State University"],
+"GT":["Georgia Institute of Technology"],
+"GVSU":["Grand Valley State University"],
+"GW":["Gardner Webb University"],
+"GWU":["Gardner Webb University"],
+"GU":["Georgetown University"],
+"HIU":["Hampton University"],
+"HMC":["Harvey Mudd College"],
+"Hopkins":["Johns Hopkins University"],
+"HU":["Howard University","Hampton University"],
+"HSU":["Humboldt State University"],
+"IC":["Ithaca College"],
+"IIT":["Illinois Institute of Technology"],
+"IPFW":["Indiana University-Purdue University Fort Wayne"],
+"ISU":["Idaho State University", "Illinois State University", "Indiana State University", "Iowa State University"],
+"IU":["Indiana University Bloomington"],
+"IUB":["Indiana University Bloomington"],
+"IUP":["Indiana University of Pennsylvania"],
+"IUPUI":["Indiana University-Purdue University Indianapolis"],
+"IWU":["Illinois Wesleyan University"],
+"IW":["Iowa Wesleyan University"],
+"JHU":["Johns Hopkins University"],
+"JMU":["James Madison University"],
+"JWU":["Johnson and Wales University"],
+"JSU":["Jackson State University"],
+"JU":["Jacksonville University"],
+"K-State":["Kansas State University"],
+"KSU":["Kansas State University", "Kennesaw State University", "Kent State University", "Kentucky State University"],
+"KU":["University of Kansas"],
```

Comment:
I'm a tad worried about some of these abbreviations, since you're dealing with a file that has global universities, and many short abbreviations can easily match other institutions. For example "KU Leuven" is a Catholic University in Leuven, in Belgium. I suspect this might be driving some false positives? Maybe there are additional features we can use when matching abbreviations?